systems/plants/@RigidBodyManipulator/doKinematics.m

```diff
@@ -1,4 +1,4 @@
-function kinsol=doKinematics(model,q,b_compute_second_derivatives,use_mex,qd)
+function kinsol=doKinematics(model,q,b_compute_second_derivatives,use_mex,qd,options)
```

Comment:
but i would support a changeover (even though it might be a little painful) to only the options struct.  the code with (true, false, true) that is around everywhere is very hard to read as well as feeling inefficient.